main.go

```diff
@@ -413,7 +416,14 @@ func decodeGMO(ctx context.Context, sDec []byte, username string) string {
 			if fort.ActivePokemon != nil {
 				newMapPokemon = append(newMapPokemon, decoder.RawMapPokemonData{Cell: mapCell.S2CellId, Data: fort.ActivePokemon})
 			}
+			// collect fort information of cell
+			if fort.FortType == pogo.FortType_CHECKPOINT {
```

Comment:
@jfberry correct me if that is the wrong approach in Golang to compare ENUM Literals :)
is there an exhaustive way to compare?

In Kotlin /Java you could do something like that:
```kotlin
when (fort.FortType) {
  pogo.FortType_CHECKPOINT -> ....
  pogo.FortType_GYM -> ...
}
```

Reply:
switch statement is what you are looking for
https://gobyexample.com/switch